logic/pulsed/pulsed_measurement_logic.py

```diff
@@ -991,6 +991,10 @@ def do_fit(self, fit_method, use_alternative_data=False, data=None):
         else:
             update_fit_data = False
 
+        if len(data) < 2 or len(data[0]) < 2 or len(data[1]) < 2:
+            self.debug.error('The data you are trying to fit does not contain enough data for a fit.')
```

Comment:
What shall it be? An error or a debug message? In any case `debug` is not an attribute of `PulsedMeasurementLogic` and so this line leads to an `AttributeError`.
I would suggest `self.log.debug(...)`

Reply:
must have been asleep in front of the keyboard, thanks